src/Arcus.Templates.ServiceBus.Topic/Program.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Arcus.Security.Core;
+using Arcus.Security.Core.Caching;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+
+namespace Arcus.Templates.ServiceBus.Topic
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            CreateHostBuilder(args)
+                .Build()
+                .Run();
+        }
+
+        public static IHostBuilder CreateHostBuilder(string[] args) =>
+            Host.CreateDefaultBuilder(args)
+                .ConfigureAppConfiguration(configuration =>
+                {
+                    configuration.AddCommandLine(args);
+                    configuration.AddEnvironmentVariables();
+                })
+                .ConfigureServices((hostContext, services) =>
+                {
+                    //#error Please provide a valid secret provider, for example Azure Key Vault: https: //security.arcus-azure.net/features/secrets/consume-from-key-vault
+                    services.AddSingleton<ISecretProvider>(serviceProvider => new CachedSecretProvider(secretProvider: null));
+
+                    services.AddServiceBusTopicMessagePump<EmptyMessagePump>("Receive-All", secretProvider => secretProvider.GetRawSecretAsync("ARCUS_SERVICEBUS_CONNECTIONSTRING"));
```

Comment:
Hm this makes me wonder, don't we have one without having to specify where the connection string comes from and we just give the name of the key? 🤔 

Might be interesting to have a `services.AddServiceBusTopicMessagePump<EmptyMessagePump>("Receive-All", "My-Secret-Name");`, what do you think?

Reply:
Hmm, but then, how do we know it's a configuration key or a (keyvault) secret key?

Reply:
It would be for secrets only in that case, discouraging folks to not use configuration.

Reply:
Yeah, makes somewhat sense. But that should be documented thorough.

Reply:
Sure, I'll make an issue for it but good to move forward here.

Reply:
https://github.com/arcus-azure/arcus.messaging/issues/48